_posts/2021-01-13-voicemeeter-an-introduction-to-mixing-audio.md

```diff
@@ -0,0 +1,385 @@
+{
+
+
+
+  Need Picture here
+
+
+
+
+}
+
+You know what we need now? Some audio to thread into these outputs! Thats what! When do we want this audio? We Want this audio yesteryear! So lets get some of that audio. In windows we need to open sound settings, it doesn't matter how we get there, whether you search in the start menu or right click the speaker icon in the task tray thing likely on the bottom right of windows unless you moved your task bar. If right clicking on the speaker icon it should be called 'Open Sound Settings' and if going through the Start Menu it will be just 'Sound Settings'. Under output we will choose our output device, this is our main output of all audio. Lets change this to 'Voicemeeter VAIO'. You may notice that if you had audio going through your listening device it likely disapeared. This is because it is now routed to the Virtual Input Voicemeeter VAIO. If you look inside Voicemeeter Banana, you should now see the audio bar boincing if you are playing audio. Do you want to hear it? Tak ea look at A1-3 and toggle the one that matches the A beside your headphones/speakers.
+
+Guided picture setup with step by step + writing explaining what to do?
+
+- Changing Windows Sound sEttings Default Audio Output Device:
+{
+
+
+
+  Picture Goes here
+
+
+
+}
+
+- Image of Virtual Inputs Section with the right toggles - Arrows and a circle with words?
+{
+
+
+
+  Picture Goes here
+
+
+
+}
+
+- Image of Virtual Inputs and Hardware Outputs with arrows directly linking the A's:
+{
+
+
+
+  Picture goes here:
+
+
+
+}
+
+- Image of Full set of Virtual Inputs, Hardware Outputs and volume bar boincing, with an arrow pointing to both the Virtual Input bar and then to the Hardware Output Bar?
+{
+
+
+
+  Picture Goes Here:
+
+
+
+}
+
+You should hear the audio now going from Voicemeeter VAIO to your headphones/speakers. If so? Congratulations! If not? Maybe please try again. If you have any issues going through this, remember I am available in our discord and would like to know any issues so I can help fix or find fixes for the and update this article based on them. (Likely will start adding FAQ to the bottom as they come in.)
+
+So what do we do with that virtual audio cable in Hardware Outputs? I'm going to guess you are looking into this as a streamer and to better answer that, lets finish up our devices. You might have noticed we haven't mentioned setting up the microphone? We are going to do that now. It doesn't matter which spot but we are going to use the first one in Hardware Inputs. If you click on 'select input device' you should get a bunch of options. I suggest the WDM version of your microphone, whatever it may be called. After selecting your microphone you may start to see the audio bar moving and it is probably picking up audio from your microphone and you may not hear it yet. That is because it has to be set to push audio to the hardware outs, just like virtual inputs had to be set to do so.
```

Comment:
Assume every person reading this site is a streamer.

Reply:
I cut this bit out in the new article. It was irrelevant fluff and the new article is more straight to the point I think?